### DISCUSSION TITLE
Zero Touch Deployment possible? (Jumpcloud, ...)

### DISCUSSION THREAD

**Discussion by shfrht** [Homebrew/discussions#4960] [Getting started] [answered]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Hi all,

in my company, we want to setup a Zero Touch Deployment on Mac devices (using Jumpcloud) and would like to install Homebrew automatically as one of the first steps. I came across several discussions and topics, including this one here: 

https://github.com/orgs/Homebrew/discussions/3199 

I would like to ask for help or suggestions regarding the issue of not being able to install Homebrew remotely as a root. Also we don't want to enter a password or pressing a key. It should be zero touch. Maybe there is a workaround that we could use or someone could state that this is definitely not possible. 

We already tried the installation with the NONINTERACTIVE=1 and CI=1 arguments.

Thanks in advance!

**Comment by tobylane** [comment]
Have you tried the two methods of using sudo non-interactively?

**Comment by SMillerDev** [comment, marked as answer]
Why not just install the pkg file?

**Reply by shfrht** [reply]
Ok thank you very much! I don't know why we didn't even think about this solution. This makes it very convenient. 

Again, thanks a lot!
